include/cocaine/rpc/slot/deferred.hpp

```diff
@@ -128,6 +128,12 @@ struct deferred {
         return *this;
     }
 
+    template <class upstream_type>
```

Comment:
Codestyle: template parameters are single capital letters (e.g. `U`) or CamelCase (e.g. `UpstreamType`). No space between `template` and an opening `<`.